bundles/framework/layerselection2/resources/css/style.css

```diff
@@ -55,11 +55,22 @@ div.layerselection2 .layer.selected {
           padding-top: 2px; }
         div.layerselection2 .layer.selected div.layer-tools div.right-tools div.object-data a {
           padding-left: 5px; }
+        div.layerselection2 .layer.selected div.layer-tools div.right-tools div.object-data div.show-metadata-tool {
+          width: 16px;
+          height: 16px;
+          background-image: url("../images/icon-info.png");
+          background-repeat: no-repeat; }
+          div.layerselection2 .layer.selected div.layer-tools div.right-tools div.object-data div.show-metadata-tool:hover {
+            background-image: url("../images/icon-info_hover.png"); }
+        div.layerselection2 .layer.selected div.layer-tools div.right-tools div.object-data div.show-filter-tool {
```

Comment:
Same here. Maybe add a flag if a tool that is added for a layer should be shown with an icon OR text. The containers for the icons could be styled here, but the actual icons should be in the bundles that add the tool.